src/routes/UserSettings/EditProfileTab/EditProfileTab.js

```diff
@@ -99,6 +171,13 @@ export default class EditProfileTab extends Component {
   updateSettingDirectly = (key, changed) => value =>
     this.updateSetting(key, changed)({ target: { value } })
 
+  updateSocialSetting = ({ key, value }) => {
```

Comment:
seems like you might be able to use/repurpose updateSettingDirectly here instead of creating a new function?

Reply:
You're right, I can! I removed `updateSocialSetting` in favor of using `updateSettingDirectly`